components/candidateTextbox.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { ApprovalType, AssessmentType } from '../types/Types';
+import Grid from '@mui/material/Grid';
+import Paper from '@mui/material/Paper';
+import parse from 'html-react-parser';
+import Pointsbox from "./pointsbox";
+
+interface approvalTexboxProp {
+  assessment: ApprovalType;
+  setAssessmentScore: (assessment: AssessmentType, newScore: number) => void;
+}
+
+const CandidateTextbox: React.FC<approvalTexboxProp> = (
+  props: approvalTexboxProp
+) => {
+  let inconsistentScoresString = '';
```

Comment:
Is this necessary for the `CandidateBox` when it uses the approved assessments from `localstorage` and there are no inconsistent scores saved there?

Reply:
Excellent point! I thought not, BUT I wanted the boxes to have tasknumber, not candidateID, soooo I just made a new one. Had we been earlier in the project I would have made a more generic solution :)